### ISSUE TITLE
Is it time to remove libming / SWF support?

### ISSUE THREAD

**Issue by ryandesign** [autotrace/autotrace#137]
autotrace uses libming if found, but since Adobe killed the SWF format years ago and libming has been unmaintained for years and has [hundreds of secutity issues](https://github.com/libming/libming/issues) filed against it, is there still any value in this or should it be removed?

What I also found strange was that there is no way to disable autotrace's SWF support:

https://github.com/autotrace/autotrace/blob/cba2237e379ab82d4c0c5d1ec86f7b69732c162f/configure.ac#L160-L176

It enables support if libming is found and does not provide a configure argument to turn it off, unlike libpng, ImageMagick, and pstoedit support which can each be disabled with a configure argument.

**Comment by lemenkov** [comment]
Yeah sonds like we should. Since it's an API-breaking change let's tag it as a future 0.40 version

**Comment by mohawk2** [comment]
Would the API be considered maintained if the relevant functions were turned into stubs that just return an error, and so not need to wait for a future 0.40?
